docs/blogs/conformance.html

```diff
@@ -0,0 +1,156 @@
+            <span class="horizontal-logo-symbol">~</span>
+            <span class="horizontal-logo-symbol">~</span>
+            <span class="edge-logo-symbol">*</span>
+        </div>
+        <div class="logo-line">
+            <span class="vertical-logo-symbol">|</span>
+            <span class="logo-text"> commonware </span>
+            <span class="vertical-logo-symbol"> </span>
+        </div>
+        <div class="logo-line">
+            <span class="edge-logo-symbol">*</span>
+            <span class="horizontal-logo-symbol">~</span>
+            <span class="horizontal-logo-symbol">+</span>
+            <span class="horizontal-logo-symbol">+</span>
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="horizontal-logo-symbol"> </span>
+            <span class="horizontal-logo-symbol">~</span>
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="horizontal-logo-symbol">+</span>
+            <span class="horizontal-logo-symbol"> </span>
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="horizontal-logo-symbol">*</span>
+            <span class="horizontal-logo-symbol">-</span>
+            <span class="edge-logo-symbol">+</span>
+        </div>
+    </div>
+    <div class="content">
+        <h1>Change is the Only Constant</h1>
+        <div class="meta">
+            <div class="author">By <a href="https://x.com/vex_0x">Ben Clabby</a></div>
+            <div class="date">December 17, 2025</div>
+        </div>
+        <p>Picture this: You and your team are scrambling to get the latest release of your project out, and in one of several hasty PR reviews, you and your fancy LLM code reviewers overlook a breaking change to the protocol. You cut the release, and during your celebratory dinner, you notice you've got 26 unread messages on Slack.</p>
+        <p>To some degree, we've all been there. Whether it be a small diff to a type's wire format or a semantic difference in operation order within a mechanism, silent breakage of functionality can consume days of your time and hurt the reputation of your project.</p>
+
+        <h2><code>conformance</code></h2>
+        <p>Detecting as many of these breaking changes as possible <i>before</i> they land in the hands of our users is critical, and at the same time, we don't trust ourselves to protect every nuanced change with manual review (nor do we expect our users to.) By defining a set of tests that validate the behavior of your implementation against a known-good reference, you can catch breaking changes before they make it into your trunk branch. Our code is constantly evolving, but we need to make sure that some things stay in-place.</p>
+
+        <p>We've introduced a new primitive, <code>conformance</code>, to offer an easy path to define and run "conformance tests." All that it takes is implementing the following trait on types that you'd like to constrain:</p>
```

Comment:
nit: creating manual cases is brittle/difficult, better to just automate with enough samples

(would be good to hit harder on the automation rationale imo)